presto-main/src/main/java/io/prestosql/metadata/MetadataManager.java

```diff
@@ -710,38 +711,17 @@ public void finishStatisticsCollection(Session session, AnalyzeTableHandle table
                 .map(layout -> new NewTableLayout(catalog, transactionHandle, layout));
     }
 
-    @Override
-    public void beginQuery(Session session, Multimap<CatalogName, ConnectorTableHandle> tableHandles)
-    {
-        for (Entry<CatalogName, Collection<ConnectorTableHandle>> entry : tableHandles.asMap().entrySet()) {
-            ConnectorMetadata metadata = getMetadata(session, entry.getKey());
-            ConnectorSession connectorSession = session.toConnectorSession(entry.getKey());
-            metadata.beginQuery(connectorSession, entry.getValue());
```

Comment:
`io.prestosql.spi.connector.ConnectorMetadata#beginQuery` is no longer being called.

Technically, it should be removed.

More importantly, I don't see yet how its role is replaced now.